tools/pepquery2/pepquery2.xml

```diff
@@ -0,0 +1,809 @@
+#end if
+  $modifications.unmodified
+  $modifications.aa
+#if $ms_params.tolerance_params.precursor_tolerance
+  -tol '$ms_params.tolerance_params.precursor_tolerance'
+#end if
+
+#if $ms_params.tolerance_params.precursor_unit
+  -tolu '$ms_params.tolerance_params.precursor_unit'
+#end if
+#if $ms_params.tolerance_params.tolerance
+  -itol '$ms_params.tolerance_params.tolerance'
+#end if
+#if $ms_params.search.frag_method
+  -fragmentMethod '$ms_params.search.frag_method'
+#end if
+#if $ms_params.search.scoring_method
+  -m '$ms_params.search.scoring_method'
+#end if
+  $ms_params.search.extra_score_validation
+#if $ms_params.search.max_charge
+  -maxCharge '$ms_params.search.max_charge'
+#end if
+#if $ms_params.search.min_charge
+  -minCharge '$ms_params.search.min_charge'
+#end if
+#if $ms_params.search.min_peaks
+  -minPeaks '$ms_params.search.min_peaks'
+#end if
+#if $ms_params.search.isotope_error
+  -ti '$ms_params.search.isotope_error'
+#end if
+#if $ms_params.search.min_score
+  -minScore '$ms_params.search.min_score'
+#end if
+#if $ms_params.search.min_length
+  -minLength '$ms_params.search.min_length'
+#end if
+#if $ms_params.search.max_length
+  -maxLength '$ms_params.search.max_length'
+#end if
+#if $ms_params.search.num_random_peptides
+  -n '$ms_params.search.num_random_peptides'
+#end if
+#if 'psm_annotation.txt' in $outputs_selected
+  -plot
+#end if
+  $fast
+  -o pepquery_output
+  | tee >(sed "s/\x1b[^m]*m//g" > log.txt) 
+#set $flist = str($outputs_selected).replace(',',' ').replace('ms_index','')
+&& for i in $flist; do for f in `find pepquery_output/*/* -name \$i`; do cat \$f >> pepquery_output/\${i}; done; done
+&& for f in `find pepquery_output/*/ -name parameter.txt`; do cp \$f pepquery_output/parameter.txt; done 
+        ]]>
+    </command>
+    <inputs>
+        <conditional name="validation"> 
+            <param name="task_type" argument="-s" type="select" label="Validation Task Type">
+                <option value="novel" selected="true">novel peptide/protein validation</option>
+                <option value="known">known peptide/protein validation</option>
+            </param>
+            <when value="novel"/> 
+            <when value="known"> 
+                 <param name="decoy" argument="-decoy" type="boolean" truevalue="-decoy" falsevalue="" checked="false" label="Identity the decoy version of the selected target protein"/>
+            </when>
+        </conditional>
+        <section name="req_inputs" title="Input Data" expanded="true">
+            <conditional name="input_type">
+                <param name="input_type_selector" argument="-t" type="select" label="Input Type" help="" >
+                    <option value="peptide">peptide</option>
+                    <option value="protein">protein</option>
+                    <option value="DNA">DNA (translate to protein sequences)</option>
+                    <!-- these are not working with pepquery version 1.6
```

Comment:
But this is version 2? Should we drop this?

Reply:
@bgruening  They are valid forms of input, but I have not implemented them.  I probably left that commented code in there in case someone wanted to make this complete.   I'll try to address your comments later in the week.  Once again thanks for the careful review.